source/includes/oai-pmh/_root.md

```diff
@@ -143,6 +187,39 @@ correct community identifier.
 [See example](https://zenodo.org/oai2d?verb=ListRecords&
 metadataPrefix=oai_datacite&set=user-cfa)
 
+
+```python
+# Fetch a couple of records from the OAI Set of the "cfa" community
+>>> records = sickle.ListRecords(metadataPrefix='oai_dc', set='user-cfa')
+>>> record = records.next()
+>>> record.sets  # ...or whatever one can do to inspect what sets a record is in
+```
+
+### Harvesting with a different metadata format
+
+```python
+# Community harvest using "marc21" metadata format
+>>> records = sickle.ListRecords(metadataPrefix='marc21', set='user-cfa')
+>>> record = records.next()
+>>> record.raw  # ...or whatever showcases the metadata format
+<...>
+```
+
+
+### Even more selective harvesting
```

Comment:
Maybe the title could be more informative if it was something like 'harvesting with multiple filters'.